rust-toolchain

```diff
@@ -1 +1 @@
-nightly-2022-01-01
+nightly-2022-09-01
```

Comment:
I think it's better to use a `rust-toolchain.toml` here, https://rust-lang.github.io/rustup/overrides.html

![image](https://user-images.githubusercontent.com/4054836/194706644-14f55a89-5031-40ed-b9a7-af9294d7f808.png)
So no need to manually run `rustup target add wasm32-unknown-unknown`  as the readme says.